docs/users.rst

```diff
@@ -6,18 +6,30 @@ View supports the creation of read only users that have access to all views and
 Adding
 ------
 
-To create a new user, firstly create the user as a postgres user yourself. Something like:
+To create a new user, firstly create the user as a Postgresql user yourself.
+
+On a Ubuntu Linux server, after connecting as root this would probably look like:
+
+.. code-block:: bash
+
+    # su postgres
+    $ psql template1
 
 .. code-block:: sql
 
     CREATE USER testreadonly with PASSWORD 'k1ngf1sher' NOCREATEDB NOSUPERUSER NOCREATEROLE;
```

Comment:
It'd be good to recommend/link to a password generator, to have stronger passwords.